src/components/MoviesSearchContainer/MoviesSearchContainer.js

```diff
@@ -16,9 +16,9 @@ const MoviesSearchContainer = ({ searchQuery }) => {
 
   return (
     <div className="movies-search-container">
-      <div className="movies-search-container__background" />
-      <div className="movies-search-container__container">
-        <div className="movies-search-container__container__add-movie">
+      <div className="container-background" />
```

Comment:
Please put the background directly in CSS using the same concepts I've mentioned (and found in articles) in the comment for separators - There's often no point in having a separate div just to display a background.